src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,26 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
+        File file;
+        try {
+            file = new File(fileName);
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file", e);
+        }
```

Comment:
doesn't it work without creating a new file?

Reply:
Works. Refactored. =)